src/components/Menu/push.vue

```diff
@@ -21,6 +21,7 @@
             this.$emit("closeMenu")
         },
         push() {
+          this.openMenu()
```

Comment:
Components with push/pull need to call openMenu and closeMenu in push/pull respectively